### PR TITLE
Adding MOL

### PR DESCRIPTION
**Pull request by shtah**
Adding Molie Network

### CODE REVIEW

File: docker/IMPLX/docker-compose.yml [hunk 1]
```diff
@@ -128,10 +128,10 @@ services:
...
 
-networks:
-  implxnetwork:
-    external: false
-    name: implxnetwork
-  proxy:
-    external: true
-    name: proxy
\ No newline at end of file
+#networks:
```
Comment (sondreb) [inline docker/IMPLX/docker-compose.yml:131]: This was perhaps not intentional?

File: docker/MOL/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      chain:
+        condition: service_healthy
+    networks:
+      - molxnetwork
+      - proxy
+
+  chain:
+    container_name: mol-chain
+    image: blockcore/node-multi:1.1.50
+    mem_limit: 6144m
```
Comment (sondreb) [inline docker/MOL/docker-compose.yml:43]: This is a lot more than is really needed, but we'll tune it later. Probably would be fine with 3GB or 4GB.

File: docker/MOL/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      - blockchain:/root/.blockcore
+    ports:
+      - 22105:22105 # Make this a public node.
+    networks:
+      - molnetwork
+
+  mongo:
+    container_name: implx-mongo
+    image: mongo:5.0.12
+    mem_limit: 3072m
```
Comment (sondreb) [inline docker/MOL/docker-compose.yml:64]: Probably enough with 2GB.

**Review by sondreb — APPROVED** [review]
Great work, thanks for doing this so I didn't have to =)

File: docker/IMPLX/docker-compose.yml (hunk 1 above)
Comment (shtah) [reply docker/IMPLX/docker-compose.yml:131]: Yes, thanks

File: docker/MOL/docker-compose.yml (hunk 2 above)
Comment (shtah) [reply docker/MOL/docker-compose.yml:43]: I made a copy of the current IMPLX settings. I will reduce

File: docker/MOL/docker-compose.yml (hunk 3 above)
Comment (shtah) [reply docker/MOL/docker-compose.yml:64]: I made a copy of the current IMPLX settings. I will reduce

**Pull request closed without merging**